pkg/mkimage-raw-efi/install

```diff
@@ -28,27 +32,36 @@ bail() {
       collect_black_box /run/INVENTORY 2>/dev/null
    fi
    echo "$*"
-   exit 1
+   $BAIL_FINAL_CMD
 }
 
 trunc() {
   rm "$1"
   touch "$1"
 }
 
+mounted_dev() {
+   local STAT
+   STAT=$(stat -c '%d' "$1" )
+   # shellcheck disable=SC2035
+   basename "$(dirname "$(cd /sys/block && grep -l '^'$(( STAT / 256 )):$(( STAT % 256 ))'$' */dev */*/dev 2>/dev/null || echo null/dev)")"
+}
+
```

Comment:
Unfortunately it does not work in my experiments: `dd: failed to open '/parts/rootfs.img': No such file or directory`. And `lrwxrwxrwx  1 0 0      13 Mar 24 17:16 rootfs.img -> '/dev/dev'$'\n''null'`.

Can we move onto something like `findmnt -n -o SOURCE --target /root` here? It works for me.

Reply:
No, I don't think we should move to that -- I'd rather find out the root cause here. What is your experiment so I can replicate it @petr-zededa ?

Reply:
>No, I don't think we should move to that

Got it. Will try to find out the problem

>What is your experiment

It comes to me inside `make installer-net&&make current&&make ACCEL=1 run-installer-net`.
I slightly modify scripts to show logs onto /dev/ttyS0 and see on console
```
dd: failed to open '/parts/rootfs.img': No such file or directory.
Installation failed. Entering shell...
```

Reply:
root cause: `grep: */*/dev: No such file or directory`

Reply:
Hey @giggsoff seems that on a 100% clean install I can repro this -- I'll fix it tomorrow

Reply:
Now it works, thank you!